config/Make.rules.Darwin

```diff
@@ -94,6 +94,9 @@ Ice[iphonesimulator]_system_libs := $(Ice[iphoneos]_system_libs)
 IceSSL[iphonesimulator]_system_libs := $(IceSSL[iphoneos]_system_libs)
 IceIAP[iphonesimulator]_system_libs := $(IceIAP[iphoneos]_system_libs)
 
+icegridadmin[macosx]_system_libs := -ledit -lncurses
+icestormadmin[macosx]_system_libs := -ledit -lncurses
```

Comment:
Is ncurses required? looking at the doc it says it is not
``
The small size (<30k), lack of dependencies (ncurses not needed!), and the free license should make this library interesting to many embedded developers.
```
https://github.com/troglobit/editline

Shouldn't this be `-leditline` instead of `-ledit -lncurses`?

Reply:
On macOS, libedit (not libeditline) and libncurses ship with the system. There is no libeditline on macos. And libedit requires libncurses on macOS.

It's possible we could use a different "edit" library on some Linux distributions. That's for another PR.